lua/telescope/pickers.lua

```diff
@@ -1013,6 +1088,7 @@ function Picker:get_status_updater(prompt_win, prompt_bufnr)
 end
 
 
+-- TODO: document what the purpose of this is
```

Comment:
returns a function that will process a element. Increase number `m` and `n` if it isnt filter (virtual text one). and basically run the file ignore check (decrease `m` again because if true we ignore the entry) and run the sorters score function (which can be anything, substr, fzy, fzf, ...)